streamlit_web.py

```diff
@@ -111,7 +125,7 @@ def retreive_data(data_retreive):
             "You will need to store keys and endpoints in the \
 environment variables")
     if not input_assignments:
-        landing_pg()
+        readme()
```

Comment:
```suggestion
        landing_pg()
```
This should probably stay as `landing_pg()` so the `interactive()` can be accessible even without an input path.